lib/nice_partials/partial.rb

```diff
@@ -23,5 +33,12 @@ def content_for(name, content = nil, options = {}, &block)
     def content_for?(name)
       @view_context.content_for?("#{name}_#{@key}".to_sym)
     end
+
+    def capture(block)
+      if block&.arity == 1
+        # Mimic standard `yield` by calling into `_layout_for` directly.
+        self.output_buffer = @view_context._layout_for(self, &block)
```

Comment:
Is this causing `capture` to be called twice per partial render? or am I mistaken?

Reply:
If there's still a plain yield in the template, then yes. It's a concession we made to move forward on this for now.

Reply:
Thanks! I think I understand it now 👍